release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -2807,6 +2807,56 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.19.1
+[id="ocp-4-19-1_{context}"]
+=== RHSA-2025:9278 - {product-title} {product-version}.1 image release, bug fix, and security update advisory
+
+Issued: 24 June 2025
+
+{product-title} release {product-version}.1, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:9278[RHSA-2025:9278] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2025:9279[RHSA-2025:9279] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.19.1 --pullspecs
+----
+
+[id="ocp-4-19-1-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a bug was triggered when adding vCenter cloud credentials for the post installation of the {ai-full} because of an invalid ConfigMap object for the cloud provider configuration. As a result, a `missing vcenterplaceholder` error was displayed. With this release, the ConfigMap data is correct, and the error is not displayed. (link:https://issues.redhat.com/browse/OCPBUGS-57384[OCPBUGS-57384])
```

Comment:
```suggestion
* Previously, when you added vCenter cloud credentials for the post installation of the {ai-full}, a bug was triggered because of an invalid `ConfigMap` object for the cloud provider configuration. As a result, a `missing vcenterplaceholder` error was displayed. With this release, the `ConfigMap` data is correct, and the error is not displayed. (link:https://issues.redhat.com/browse/OCPBUGS-57384[OCPBUGS-57384])
```